pacai/bin/pacman.py

```diff
@@ -453,9 +452,12 @@ def readCommand(argv):
     elif options.debug:
         updateLoggingLevel(logging.DEBUG)
 
-    # Fix the random seed.
-    if options.fixRandomSeed:
-        random.seed(FIXED_SEED)
+    # If seed value is not entered generate a random seed value
+    if options.seed:
+        random.seed(options.seed)
+    else:
+        seed = random.randint(0, 2**32)
+        logging.debug('Seed value: ' + str(seed))
```

Comment:
We want to log the seed always.
Always mirror pacman and capture as much as you reasonably can.